.github/workflows/docker-bitnami-release.yaml

```diff
@@ -0,0 +1,73 @@
+name: Docker Bitnami Release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-rc[0-9]+'
+
+
+jobs:
+  docker-release:
+    name: Docker Image Release
+    strategy:
+      matrix:
+        platform: [ "ubuntu-24.04" ]
+        jdk: ["17"]
+    runs-on: ${{ matrix.platform }}
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up JDK ${{ matrix.jdk }}
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.jdk }}
+          distribution: "zulu"
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2.12.0
+      - name: Get project version
+        id: get_project_version
+        run: |
+          project_version=$(./gradlew properties | grep "version:" | awk '{print $2}')
+          echo "PROJECT_VERSION=${project_version}" >> $GITHUB_OUTPUT
+
+      - name: Build TarGz
+        run: |
+          ./gradlew -Pprefix=automq-${{ github.ref_name }}_ --build-cache --refresh-dependencies clean releaseTarGz
+
+      # docker image release
+      - name: Cp TarGz to Docker Path
+        run: |
+          cp ./core/build/distributions/automq-${{ github.ref_name }}_kafka-${{ steps.get_project_version.outputs.PROJECT_VERSION }}.tgz ./container/bitnami
+      - name: Determine Image Tags
+        id: image_tags
+        run: |
+          LATEST_TAG=$(git tag --sort=-v:refname | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | head -n 1)
+          echo "LATEST_TAG=${LATEST_TAG}"
+          if [ "$LATEST_TAG" == "${{ github.ref_name }}" ]; then
+            echo "tags=${{ secrets.DOCKERHUB_USERNAME }}/automq-bitnami:${{ github.ref_name }}, ${{ secrets.DOCKERHUB_USERNAME }}/automq-bitnami:latest" >> $GITHUB_OUTPUT
+          else
+            echo "tags=${{ secrets.DOCKERHUB_USERNAME }}/automq-bitnami:${{ github.ref_name }}" >> $GITHUB_OUTPUT
```

Comment:
We should still use the name automq and only change the tag to ${{ github.ref_name }}-bitnami